compiler/rustc_middle/src/ty/sty.rs

```diff
@@ -1483,22 +1483,35 @@ pub enum RegionKind {
     ReErased,
 }
 
-#[derive(Copy, Clone, PartialEq, Eq, Hash, TyEncodable, TyDecodable, Debug, PartialOrd, Ord)]
+#[derive(
```

Comment:
This formatting is really obnoxious - is there a way to configure rustfmt to full up each line with arguments before moving to the next line?